src/backends/runtime/eval.ml

```diff
@@ -13,36 +13,33 @@ let update x = RuntimeEnv.add x
 let lookup x state =
   try Some (RuntimeEnv.find x state) with Not_found -> None
 
-exception PatternMatch of Location.t
```

Comment:
Was there a particular reason you removed the location information from the exception here?

Reply:
I did that because in the ImpEff syntax, the pattern data type contains a location ("plain pattern located"), but in the ExEff syntax it does not. So where that location was retrieved from the ImpEff patterns in the original runtime backend, that is not possible when taking ExEff as an input. As this exception is only thrown when pattern matching on the pattern data type, I figured I could just change its data type not to take the location info anymore.